test/image/mocks/error_bar_style.json

```diff
@@ -72,16 +72,14 @@
     "value": 0.1,
     "color": "#85144B",
     "thickness": 1.5,
-    "width": 3,
-    "opacity": 1
```

Comment:
just for my own learning, what happened to `"opacity"`?

Reply:
`opacity` can also be specified by using the alpha channel of the `color` attribute, which defaults to full-opacity, so the same image result can be achieved without explicitly setting the opacity. 

I guess for the purposes of this test it might make sense to include it explicitly though.

Reply:
I don't know why we chose to deprecate `opacity` here though and not elsewhere, maybe @archmoj has an idea

Reply:
In some places - particularly where multiple colors or multiple objects are involved - there are [many different effects](https://chart-studio.plotly.com/~alex/455/four-ways-to-change-opacity-of-scatter-markers.embed) you can achieve depending on exactly where you set the opacity

<img width="749" alt="Screenshot 2024-10-08 at 10 15 34" src="https://github.com/user-attachments/assets/5e198a4c-c1a9-40fe-b3fe-48ac1ecdcc82">

Here though I guess there's nothing you can't achieve just with the opacity of the color.

Reply:
That makes perfect sense -- thanks @alexcjohnson .